src/QLContext.actor.cpp

```diff
@@ -35,15 +35,19 @@ Future<DataValue> IReadContext::toDataValue() {
 	return getRecursiveKnownPresent(Reference<IReadContext>::addRef(this));
 }
 
-ACTOR Future<Void> DocTransaction::commitChanges(Reference<DocTransaction> self, std::string docPrefix) {
+ACTOR static Future<Void> commitChangesActor(Reference<DocTransaction> self, std::string docPrefix) {
```

Comment:
`static` is no longer necessary unless you do want to hide the symbol in `.o` file.

Reply:
Yeah, this is local to just this file.

Reply:
Oh yeah I missed that...